packages/cli/src/controller/migrate/schema/migrate-schema.controller.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020-2024 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+/**
+ *  Migrate a subgraph project schema to subquery schema file
+ * @param network network family
+ * @param inputPath file path to subgraph.yaml
+ * @param outputPath file path to project.ts
+ */
+export async function migrateSchema(inputPath: string, outputPath: string) {
+  //TODO
```

Comment:
Maybe here for now, we could log instructions as to what to do. Can have a link to some docs etc.